### PR TITLE
Added Domain to setup wizard

### PR DESCRIPTION
**Pull request by neilLasrado**

### CODE REVIEW

File: erpnext/public/js/setup_wizard.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ function load_erpnext_slides() {
 					placeholder:__('e.g. "Build tools for builders"'), reqd:1},
 				{fieldname:'bank_account', label: __('Bank Account'), fieldtype:'Data',
 					placeholder: __('e.g. "XYZ National Bank"'), reqd:1 },
+				{fieldname:'domain', label: __('Domain'), fieldtype:'Select',
+					options:["Distribution", "Manufacturing", "Retail", "Services", "Education"], reqd:1},
```
Comment (anandpdoshi) [inline erpnext/public/js/setup_wizard.js]: Add "Other" option too in both Company and this list. Also, did you confirm if this translates?

**Comment by neilLasrado** [comment]
@anandpdoshi Added Other in Domain. I have checked the translation. Works fine.

File: erpnext/setup/setup_wizard/setup_wizard.py [hunk 2]
```diff
@@ -66,16 +66,20 @@ def create_fiscal_year_and_company(args):
...
 			'company_name':args.get('company_name').strip(),
 			'abbr':args.get('company_abbr'),
 			'default_currency':args.get('currency'),
 			'country': args.get('country'),
 			'chart_of_accounts': args.get(('chart_of_accounts')),
+			'domain': args.get('domain')
 		}).insert()
-
+		
+		if "schools" in frappe.get_installed_apps():
```
Comment (anandpdoshi) [inline erpnext/setup/setup_wizard/setup_wizard.py]: This should be part of Schools app!

**Comment by anandpdoshi** [comment]
Removed schools part and merged manually

**Pull request closed without merging**